vue/dynamicforms/src/components/api_consumer/form-consumer/api-one-shot.spec.ts

```diff
@@ -0,0 +1,22 @@
+import MockAdapter from 'axios-mock-adapter';
+
+import apiClient from '../../util/api-client';
+
+import FormConsumerApiOneShot from './api-one-shot';
+import data from './api-one-shot.spec.json';
+
+const mock = new MockAdapter(apiClient);
+
+describe('Api One Shot', () => {
+  it('Create', async () => {
+    mock.onGet('/test.componentdef').reply(200, data);
+
+    const consumer = FormConsumerApiOneShot({ url: '/test' });
+    expectTypeOf(consumer).toEqualTypeOf(FormConsumerApiOneShot);
+
+    // wait for consumer to establish before resetting mock
+    // eslint-disable-next-line no-promise-executor-return
+    await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Seveda je to bedastoča, ki jo PRAVILNO rešiš z await...

Reply:
m islim, da ne bo pomote: await apiconsumeroneshot...

Reply:
Ne bo slo. Await apiconsumeroneshot bo cakalo v nedogled. Predlog za drugacno resitev?

Reply:
https://stackoverflow.com/questions/55660127/how-to-write-a-unit-test-to-check-if-modal-shows-up-after-clicking-some-link

ali pa emittaš en event, ko se zgodi, kar ti pač misliš, da se mora zgodit. Najverjetneje je dovolj, če awaitaš nextTick.

Nikakor pa 500ms delay ni pravilen način za to. Se bodo unit testi izvajali ure...